web/spacewalk-web.spec

```diff
@@ -68,6 +68,18 @@ BuildRequires:  nodejs
 %description -n susemanager-web-libs
 This package contains Vendor bundles needed for spacewalk-web
 
+%package -n susemanager-web-libs-debug
+Summary:        Vendor bundles for spacewalk-web debug files
+License:		0BSD and BSD-3-Clause and LGPL-3.0-or-later and MIT and MPL-2.0
+Group:          Applications/Internet
+
+BuildArch:      noarch
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+Requires:       susemanager-web-libs
```

Comment:
It's really weird how we mix packages names here, but not your fault.

Still, we should consider creating a macro with the content `susemanager-web`, so we could use it everywhere.

Reply:
I'll leave that to you, I have no idea what the names actually should be and don't want to introduce an update mess by changing package names the wrong way.